core/common/src/test/BackgroundMapSettings.test.ts

```diff
@@ -51,32 +49,13 @@ describe("BackgroundMapSettings", () => {
 
       const expectedSettings = BackgroundMapSettings.fromJSON(expected);
       expect(settings.equals(expectedSettings)).to.be.true;
-
-      // Check synch through base map layer.
-      const mapLayer = MapLayerSettings.fromMapSettings(settings);

```

Comment:
Make certain the deleted tests are replaced with (1) versions that confirm the old persistent JSON representation still behaves as previously and (2) the equivalent new version behaves the same.

Reply:
For this particular test, there isnt much we cant test right?  Should I keep testing old JSON (i.e `{ providerName: "BingProvider" }`) and make sure it has no effect on BackgroundMapSettings anyomore?